packages/react-components/src/atoms/Paragraph.tsx

```diff
@@ -14,23 +14,31 @@ const primaryStyles = css({
   lineHeight: `${24 / 18}em`,
 });
 
+const smallStyles = css({
```

Comment:
This is not in the design spec o.O are you sure it's not `Caption` (13.6)

Reply:
Maybe it is :) checking.